lib/BuildSystem/SwiftTools.cpp

```diff
@@ -175,6 +179,10 @@ class SwiftCompilerShellCommand : public ExternalCommand {
   /// Whether the sources are part of a library or not.
   bool isLibrary = false;
 
+  /// Contains the "began" messages produced by swiftc's parseable-output
+  /// mapped to their pid.
+  std::unordered_map<int32_t, SwiftCompilerOutputMessage> outputmap;
```

Comment:
Is there a need to store the entire message? What do we use the rest of it for?

Also, camelCase please.